### ISSUE TITLE
The task starts but don't run any test

### ISSUE THREAD

**Issue by HectorBritoDev** [calebporzio/better-phpunit#66]
When i run the test the task starts but no test is run. I always see something like that

```
> Executing task in folder SGI: c:/laragon/www/SGI/vendor/bin/phpunit c:/laragon/www/SGI/tests/Feature/ExampleTest.php --filter '^.*::testBasicTest$' <


Terminal will be reused by tasks, press any key to close it.
```

No info about the test or even an error in the middle. What can i do?

I'm using

Version: 1.28.0-insider (user setup)
Commit: 7fcd81cd0f88bb2040f249b09cf6bdd83d9ea2ef
Date: 2018-09-17T09:37:55.086Z
Electron: 2.0.9
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

**Comment by marcelloh** [comment]
I have exactly the same, but here, at the end, it tries to open a file (what file) and suggest it needs to be opens by some office software application.

**Comment by EmmaPurple** [comment]
Same here, but while running a test a new terminal is startet, but i can't tell what it does. I believe there are only two or three rows of text, but it closes itselfs immediately. I tried to get a screenshot for some time, but its to fast, even while the CPU max multiplicator was set as low as possible...

and sorry for my bad english, haven't used it for many, many years...

**Comment by EmmaPurple** [comment]
Perhaps a solution (in my case it worked)

Add the user Setting for the terminal, in my case (Windows) 
"terminal.integrated.shell.windows": "C:\\WINDOWS\\System32\\cmd.exe",

Credits go to Adam Matysiak on https://laracasts.com/series/visual-studio-code-for-php-developers/episodes/10
#comment-3699175456

**Comment by gjrdiesel** [comment]
Confirmed with @EmmaPurple in my case I used git bash & laragon on windows

My settings.json looks like
```JSON
...
"terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe",
"php.validate.executablePath": "C:\\laragon\\bin\\php\\php-7.2.11-Win32-VC15-x64\\php.exe",
...
```

Also for those of you new to visual studio code you can edit settings json by hitting `Ctrl+Shift+P` type typing `settings.json`.

I now have proper output but no color (Windows 🤷‍♂️ )

**Comment by Pen-y-Fan** [comment]
All these problems appear to be windows related. 

I have posted a fix, which is awaiting approval: see https://github.com/calebporzio/better-phpunit/blob/3e7d64fb65bfb4de9d4e92868e3e7ecd947358f5/src/phpunit-command.js

Please, can you:

1. Download this phpunit-command.js file from the link above?
Close VS Code
2. Backup the phpunit-command.js file in C:\Users\yourusername \vscode\extensions\calebporzio.better-phpunit-1.5.0\src
3. Replace it with the download file from 1.
4. Reopen VS Code
5. Run some tests.

Please let us know how you get on. I hope if other Windows users confirm the fix works from them then the fix will be approved.

**Comment by calebporzio** [comment]
Should be fixed in v1.5.1. Re-open if it's not. Thanks.
